meson_cpu/arm/meson.build

```
@@ -51,8 +51,15 @@ ASIMDFHM = mod_features.new(
   args: {'val': '-march=armv8.2-a+fp16fml', 'match': '-march=.*', 'mfilter': '\+.*'},
   test_code: files(source_root + '/numpy/distutils/checks/cpu_asimdfhm.c')[0]
 )
+## Scalable Vector Extensions (SVE)
+SVE = mod_features.new(
+  'SVE', 8,
```

```suggestion
  'SVE', 8, implies: [ASIMDFHM, ASIMDDP],
```
As far as I am aware, CPUs with (SVE) also support Advanced SIMD Floating-point Half Precision (ASIMDFHM), Advanced SIMD Dot Product (ASIMDDP), and beyond.

---

Ah, thanks, you're half right. SVE implies ASIMDFHM, but not ASIMDDP. Will update :+1:

---

Thank you for the update.